app/lib/features/pins/widgets/pin_list_item.dart

```diff
@@ -56,15 +56,16 @@ class _PinListItemState extends ConsumerState<PinListItem> {
         child: ListTile(
           key: Key(pin.eventIdStr()), // FIXME: causes crashes in ffigen
           leading: Icon(isLink ? Atlas.link_chain_thin : Atlas.document_thin),
-          title: Text(pin.title()),
+          title: Text(
+            pin.title(),
+            softWrap: false,
+            overflow: TextOverflow.ellipsis,
+          ),
```

Comment:
There is, if we use ```sizer``` or similar responsive package which shrinks up the font size depending on device pixel density. I'd rather put that in separate PR where we do overall refactor of responsiveness cause we need to take care of some breakpoints e.g. Desktop where font size gets incredibly large, if its based on ```dp```.

Reply:
But sizer would not quite do what I mean. If you shrank because of pixeldensity large text would still be cut. I am talking about reducing the font-size depending on available-space-vs-length-of-text ... but that might be a bit more complicated. Either way, future PR work